include/dfsinternal.h

```diff
@@ -52,6 +55,30 @@ typedef struct dfs_open_file_s
     uint32_t cart_start_loc;
 } dfs_open_file_t;
 
+/** @brief DFS File Entry */
+typedef struct dfs_file_entry_s {
+    /** @brief Hash of file path */
+    uint32_t path_hash;
+    /** @brief Top 12 bits: length; lowest 20 bits: offset */
+    uint32_t path_ofs;
+    /** @brief Data offset for file */
+    uint32_t data_ofs;
+    /** @brief Data length for file */
+    uint32_t data_len;
+} dfs_file_entry_t;
+
+/** @brief Data for DFS file lookup */
+typedef struct dfs_file_lookup_s {
+    /** @brief Number of files */
+    uint32_t num_files;
+    /** @brief Size of path buffer */
```

Comment:
Same for this, a reader has no clue what a "path buffer" is.

Reply:
This member is no longer present here